### PR TITLE
Documentation for `silenceTimeout` in `<Record/>`

### PR DESCRIPTION
**Pull request by nashley**

### CODE REVIEW

File: voice/bxml/verbs/record.md [hunk 1]
```diff
@@ -17,9 +17,10 @@ The Record verb allows a segment of audio to be recorded during a call. At the e
...
 | maxDuration                  | (optional) Maximum length of recording (in seconds). Max 10800 (3 hours). Default value is 60.                                                                                                                         |
+| silenceTimeout                  | (optional) Length of silence after which to end the recording (in seconds). Max is equivalent to the maximum `maxDuration` value. Default value is 0, which means disabled.                                                                                                                         |
 | fileFormat                   | (optional) The audio format that the recording will be saved as: `mp3` or `wav`.  Default value is `wav`.                                                                                                              |
 
-The caller can press one of the `terminatingDigits` to stop the recording.
+To stop the recording, the caller can press one of the `terminatingDigits` or stop talking for `silenceTimeout` seconds.
```
Comment (nashley) [inline voice/bxml/verbs/record.md]: This terminology was already being used, but should `caller` be `user` instead?

File: voice/bxml/verbs/gather.md [hunk 2]
```diff
@@ -4,17 +4,17 @@ The Gather verb is used to collect digits for some period of time.
...
+| password          | (optional) The password to send in the HTTP request to `gatherUrl`.                                                                                                                                                                                                                                                                             |
+| tag               | (optional) A custom string that will be sent with this and all future callbacks unless overwritten by a future `tag` attribute or cleared.<br><br>May be cleared by setting `tag=""`<br><br>Max length 256 characters.                                                                                                                          |
+| terminatingDigits | (optional) When any of these digits are pressed, it will terminate the Gather. Default behavior is disabled. This feature can also be explicitly disabled with `""`.                                                                                                                                                                            |
```
Comment (n-miles) [inline voice/bxml/verbs/gather.md]: I would just say "Default: none (`""`)"

File: voice/bxml/verbs/record.md (hunk 1 above)
Comment (n-miles) [reply voice/bxml/verbs/record.md]: Yes

File: voice/bxml/verbs/record.md [hunk 3]
```diff
@@ -15,11 +15,12 @@ The Record verb allows a segment of audio to be recorded during a call. At the e
...
-| terminatingDigits            | (optional) When pressed, this digit will terminate the recording. Default value is `“#”`.                                                                                                                              |
+| terminatingDigits            | (optional) When pressed, this digit will terminate the recording. Default value is `“#”`. This feature can be disabed with `""`.                                                                                       |
 | maxDuration                  | (optional) Maximum length of recording (in seconds). Max 10800 (3 hours). Default value is 60.                                                                                                                         |
+| silenceTimeout               | (optional) Length of silence after which to end the recording (in seconds). Max is equivalent to the maximum `maxDuration` value. Default value is 0, which means disabled.                                            |
```
Comment (ghost) [inline voice/bxml/verbs/record.md]:
```suggestion
| silenceTimeout               | (optional) Length of silence after which to end the recording (in seconds). Max is equivalent to the maximum `maxDuration` value. Default value is 0, which means no timeout.                                            |
```

File: voice/bxml/verbs/gather.md (hunk 2 above)
Comment (nashley) [reply voice/bxml/verbs/gather.md]: I had typed up a reply, but I guess I didn't save it.
Basically, the established convention is `Default value is $VALUE`, and I'd like to be consistent.
Should I change all of the existing default messages?

Comment (n-miles) [reply voice/bxml/verbs/gather.md]: I don't understand your point. The message here doesn't say `Default value is $VALUE`. What I suggested is closer to that I think.

Comment (nashley) [reply voice/bxml/verbs/gather.md]: Hmmm. How about this?
```
... Default value is `""`, which disables this feature.
```

Comment (n-miles) [reply voice/bxml/verbs/gather.md]: Sure

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4085-silenceTimeout.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Review by jmulford-bw — APPROVED** [review]
Approving from a DX perspective 👍

**Review by n-miles — APPROVED** [review]

**Pull request merged**